### PR TITLE
Support Node v4.0.0 in newman

### PR DESCRIPTION
**Pull request by czardoz**
Travis is green which is a good thing. Two things remain to be tested, 

1. The changes to SugarJS in the sandbox
2. Installation on Windows

### CODE REVIEW

**Comment by czardoz** [comment]
Just tested the installation on windows

File: src/responseHandlers/TestResponseHandler.js [hunk 1]
```diff
@@ -91,7 +91,7 @@ var TestResponseHandler = jsface.Class(AbstractResponseHandler, {
         sweet += "for(p in sugar.array)  {if(p==='create'){Array.create=sugar.array.create} else{Array.prototype[p]= sugar.array[p];}}";
         sweet += "for(p in sugar.string) String.prototype[p]  = sugar.string[p];";
         sweet += "for(p in sugar.date)  {if(p==='create'){Date.create=sugar.date.create} else{Date.prototype[p]= sugar.date[p];}}";
-        sweet += "for(p in sugar.funcs)  Function.prototype[p]= sugar.funcs[p];";
+        sweet += "for(p in sugar.funcs) Object.defineProperty(Function.prototype, p, Object.getOwnPropertyDescriptor(sugar.funcs, p));";
```
Comment (shamasis) [inline src/responseHandlers/TestResponseHandler.js]: These will not work on older node. I think it must be checked before proceeding and then choose either.

File: scripts/postinstall.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    }
+    else {
+        npm.commands.install(['jsdom@6.3.0'], function (err) {
+            if (err) {
+                console.log('Could not install "jsdom"');
+                throw err; // Will cause the script to exit with a non zero code
+            }
+        });
+    }
+});
```
Comment (shamasis) [inline scripts/postinstall.js]: Add newline EOF?

**Comment by shamasis** [comment]
@czardoz - I think you should have a look at #214 and its corresponding PR #213 regarding sugar functions before merging this. Maybe that PR can be reviewed and added to develop and changes made here accordingly?

File: scripts/postinstall.js (hunk 2 above)
Comment (czardoz) [inline scripts/postinstall.js]: Fixed

**Comment by czardoz** [comment]
@shamasis cool, I've merged in that PR, but the test has been added here.

File: src/responseHandlers/TestResponseHandler.js [hunk 3]
```diff
@@ -27,7 +27,7 @@ var TestResponseHandler = jsface.Class(AbstractResponseHandler, {
     throwErrorOnLog: false,
 
     main: function () {
-        jsdom.env("<html><body></body></html>", function (err, window) {
+        jsdom.env("<html <body></body></html>", function (err, window) {
```
Comment (czardoz) [inline src/responseHandlers/TestResponseHandler.js]: Needs to be fixed

**Pull request merged**